tests/jest/unit/modules/kit-elements-defaults/assets/js/editor/utils.js

```diff
@@ -0,0 +1,64 @@
+export class MockFetch {
+	constructor( baseURL ) {
+		this.baseURL = baseURL;
+		this.mocks = [];
+
+		window.fetch = this.#makeFetchInstance();
```

Comment:
Feels a little weird that this is in the constructor, what do you think?

Reply:
Yup.
Extracted it